Makefile.module

```diff
@@ -1,10 +1,54 @@
-TAG ?= ghcr.io/USER/REPO:0.0.0
+URL ?= local-triton-build
+SEMVER ?= 0.0.0
+TAG ?= $(URL)$(SUFFIX):$(SEMVER)
 
-%.sh: bin/%.sh.envsubst
+
+# Shell scripts built with environment variable substitution should be declared .PHONY, because
+# they need to be rebuilt even if the only thing that's changed is the value of $(TAG). We declare
+# them in separate rules because one rule for `%.sh` doesn't work with .PHONY.
+.PHONY: viam-mlmodelservice-triton.sh first_run.sh
+
+first_run.sh: bin/first_run.sh.envsubst
+	cat $< | TAG=$(TAG) DOLLAR=$$ envsubst > $@
+	chmod +x $@
+
+viam-mlmodelservice-triton.sh: bin/viam-mlmodelservice-triton.sh.envsubst
 	cat $< | TAG=$(TAG) DOLLAR=$$ envsubst > $@
 	chmod +x $@
 
+
+# Docker images. These should be phony because you can't see the files that get built.
+.PHONY: images image-jetpack5 image-jetpack6 image-cuda
+images: image-jetpack5 image-jetpack6 image-cuda
+
+image-jetpack5: SUFFIX ?= /jetpack5
+image-jetpack5:
+	docker build . -t $(TAG) -f etc/docker/Dockerfile.triton-jetpack-focal
+
+image-jetpack6: SUFFIX ?= /jetpack6
+image-jetpack6:
+	docker build . -t $(TAG) -f etc/docker/Dockerfile.nvcr-triton-containers --build-arg JETPACK=1
+
+image-cuda: SUFFIX ?= /cuda
+image-cuda:
+	docker build . -t $(TAG) -f etc/docker/Dockerfile.nvcr-triton-containers
+
+
+# The module itself.
 module.tar.gz: viam-mlmodelservice-triton.sh first_run.sh
 	tar czf $@ $^
 
-.PHONY: module.tar.gz
```

Comment:
The tarball isn't really phony (need to rebuild every time): the .sh files inside it are! I wonder if this had been a bug and I just never noticed because I wasn't building the entire module locally very often. :eyes: